Technical/Performance-Benchmarking/performance-benchmarking-of-arangodb-single-instance-vs-split-databases-vs-clustering.md

```diff
@@ -0,0 +1,2030 @@
+- ****Split Databases Pod Allocation:**** Targeted at balancing load across instances, ideal for read-heavy applications requiring higher availability.
+- ****Clustering Pod Allocation:**** Designed for maximum performance and fault tolerance, suitable for write-heavy workloads and applications demanding high availability.
+
+Each pod configuration was meticulously planned to align resource allocation with expected load, ensuring each pod type (TMS, CRSP, TP, TADP, Rules, Redis, Nats, Arango) was optimized for the specific demands of the deployment.
+
+
+<span style="color:olive; font-size: 18px;">****Single Instance Pod Allocation****</span>
+| TMS | CRSP | TP | TADP | Rules | Redis | Nats | Arango |
+|-----|------|----|------|-------|-------|------|--------|
+| 80  | 155  | 440| 385  |   15  |  60   |  12  |   1    |
+
+
+<span style="color:maroon; font-size: 18px;">****2 Split Databases Pod Allocation****</span>
+| TMS | CRSP | TP | TADP | Rules | Redis | Nats | Arango |
+|-----|------|----|------|-------|-------|------|--------|
+| 200 | 190  | 500| 460  |    20 |   60  |  18  |   2    |
+
+<span style="color:orange; font-size: 18px;">****4 Split Databases Pod Allocation****</span>
+| TMS | CRSP | TP | TADP | Rules | Redis | Nats | Arango |
+|-----|------|----|------|-------|-------|------|--------|
+| 100 | 150  | 400| 400  |    16 |   60  |  12  |   4    |
+
+<span style="color:navy; font-size: 18px;">****Clustering Pod Allocation****</span>
+| TMS | CRSP | TP | TADP | Rules | Redis | Nats | Arango |
+|-----|------|----|------|-------|-------|------|--------|
+| 100 | 150  | 400| 400  |   15  |  60   |  12  |   12   |
+
+
+<span style="color:green; font-size: 24px;">****Benchmarking Criteria****</span>
+
+The criteria for the benchmarks included several key performance indicators:
+
+- ****Throughput:**** Measured in transactions per second (FTPS), to determine the raw processing power of each configuration.
+- ****Latency:**** The time taken for a single transaction to be processed, indicating responsiveness.
+- ****Concurrency:**** The ability to handle multiple simultaneous operations, which reflects on the robustness of the system under peak loads.
+- ****Resource Utilization:**** Monitoring of CPU, memory, and I/O usage to assess the efficiency of the configuration.
+- ****Stability:**** Monitored the consistency of performance over extended periods.
+- ****Scalability:**** Evaluated the ease and impact of scaling each configuration on performance.
+
+
+*****
+
+<span style="font-size: 34px;">**Results and Analysis**</span>
+
+<span style="color:green; font-size: 24px;">****Performance Metrics****</span>
+
+<span style="color:olive; font-size: 18px;">****Single Instance****</span>
+
+****Jmeter ingestion rate:**** 2859 MPS
+
+****Total messages**** 168 000
+
+****FTPS:**** 2240
+
+<span style="color:blue;">**Processors**</span>
+
+| Results    | TMS       | CRSP       | TP         | TADP      |
```

Comment:
The tabulated results below need to be explained with some kind of analysis. At least the row and column headings need to be clarified so that the reader can know what they're looking at.

Reply:
Added